cigaradvisor/styles/styles.css

```diff
@@ -23,14 +23,63 @@
   --nav-height: 64px;
  }
 
-body {
+/* body {
```

Comment:
this has to stay, its how Franklin works.

Reply:
reverted.